server/src/metadata/relation-metadata/relation-metadata.service.ts

```diff
@@ -126,20 +131,26 @@ export class RelationMetadataService extends TypeOrmQueryService<RelationMetadat
         description: record.toDescription,
         icon: record.toIcon,
         isCustom: true,
-        targetColumnMap: {},
+        targetColumnMap: toIsCustom
+          ? {
+              value: createCustomColumnName(record.toName),
```

Comment:
so we still need to use targetColumnMap? I thought the plan was to deprecate it?

Reply:
Didn't know we plan to deprecate it, we can sill change that later as it's use that way everywhere